### PR TITLE
Fix a typo.

### PR DESCRIPTION
**Pull request by csukuangfj**

### CODE REVIEW

File: articles/100_ament.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ If the resulting artifacts are installed into `/usr`, for example, it might not
 
 [ament_tools](https://github.com/ament/ament_tools) is a Python package which provides the command line tool `ament` to build, test, install, and uninstall packages.
 It is similar to [catkin_tools](https://github.com/catkin/catkin_tools) and builds each package in a workspace in topological order.
-Support for different build systems is integrated through extension points which allows to contribute support for other build types without change the ament tool itself.
+Support for different build systems is integrated through extension points which allows to contribute support for other build types without changing the ament tool itself.
```
Comment (sloretz) [inline articles/100_ament.md]: as long as you're here, I think the words `to contribute` can be deleted as well

Comment (clalancette) [reply articles/100_ament.md]: Heh, I just changed that myself.  Not sure if you just want to remove "to contribute", or if you like my "allows the implementation of".  I'm fine either way.

Comment (sloretz) [reply articles/100_ament.md]: Reads fine to me!

**Review by sloretz — APPROVED** [review]

**Review by mikaelarguedas — APPROVED** [review]

**Comment by dirk-thomas** [comment]
"build types" are imo not "implemented". I think saying that a build type like `cmake` is "supported" is more precise. Therefore I would suggest:

> which allows to support other build types without changing the ament tool itself.

**Comment by dirk-thomas** [comment]
> "build types" are imo not "implemented". I think saying that a build type like `cmake` is "supported" is more precise. Therefore I would suggest:

> > which allows to support other build types without changing the ament tool itself.

@sloretz @mikaelarguedas Do you have any comment on this?

**Comment by dhood** [comment]
I agree with you @dirk-thomas . Would just change the english to "Support for different build systems is integrated through extension points which allows support for other build types to be added without changing the ament tool itself."

**Comment by mikaelarguedas** [comment]
> mikaelarguedas Do you have any comment on this?

The current state of this PR seems fine to me:
"Support for different build systems is integrated through extension points which allows to contribute support for other build types without changing the ament tool itself."

@dhood'd suggestion is fine as well it just changes "which allows **to contribute** support for other build types" to "which allows support for other build types **to be added**".

I dont think it deserves to be changed but maybe it's more english.

**Comment by sloretz** [comment]
I'm happy with either "supported" or "implemented"

**Comment by sloretz** [comment]
I'm happy with either "supported" or "implemented".

**Comment by dirk-thomas** [comment]
I updated it to:

> ... which allows support for other build types to be added without changing the ament tool itself.

I will leave it up to someone else to squash and merge this. :+1: from me.

**Review by sloretz — APPROVED** [review]

**Comment by dirk-thomas** [comment]
@csukuangfj Thank you for your attention to detail.

**Pull request merged**